vagrant-post-script/install_docker.sh

```diff
@@ -13,8 +15,9 @@ sudo apt-key fingerprint 0EBFCD88 || { echo "[Vagrant] ###################     F
 sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu bionic stable"
 
 sudo apt-get update > /dev/null
-sudo apt-get install -qq -y docker-ce=5:18.09.4~3-0~ubuntu-bionic || { echo "[Vagrant] ###################     Installation error, exiting"; exit; }
+sudo apt-get install -qq -y "docker-ce=5:${DOCKERVERSION}~3-0~ubuntu-bionic" || { echo "[Vagrant] ###################     Installation error, exiting"; exit; }
```

Comment:
Instead of setting the Docker version in a variable, can't we use Dependabot to automatically open PRs with updates?